style.css

```diff
@@ -0,0 +1,4 @@
+/* styles for the project */
```

Comment:
- [ ] Please remove commented code from your project. It is widely considered a bad industry practice. This helps keep your code clean. You can read me on why you shouldn't commit commented out code [here](https://kentcdodds.com/blog/please-dont-commit-commented-out-code).

Reply:
- Noted 🤝